admin-dev/themes/new-theme/js/pages/order/create.js

```diff
@@ -61,10 +61,22 @@ function refreshCart() {
   orderPageManager.refreshCart();
 }
 
+/**
+ * proxy to allow other scripts within the Create Order page to refresh cart
+ */
+function addCreatedCartRuleToCart(cartRuleId) {
+  if (orderPageManager === null) {
+    console.log('Error: Cannot apply created cart rule to the current cart as orderPageManager is not available');
```

Comment:
It's normal to let console.log?

Reply:
It's the same in other methods of this file

I guess it's for helping devs to debug without throwing an error

Reply:
@tegbessou if I'm not mistaken, `console.log`s are removed at build time in production environment.